### DISCUSSION TITLE
Show after output?

### DISCUSSION THREAD

**Discussion by MACscr** [tighten/lambo#177] [Ideas]
I would like to see the output of the after bash script or have a way to display some data. Id like to use that opportunity auto create a user account in the laravel app and then display that randomly generated password. Right now you wouldnt be able to see it.

**Comment by jonsugar** [comment]
Hi @MACscr,

You can show the output of your after script by using the `-v` (verbose) flag. 

For example:

```
lambo -v new foo
```


which will output something similar to:

<img width="703" alt="ScreenShot 2022-08-12 at 12 43 42" src="https://user-images.githubusercontent.com/1974648/184348236-eda6995a-830e-4f40-bf01-8b8e3a944604.png">

Notice it will also show the output of all the commands lambo issues which might be too much 🤔.

Does this give you what you need?
